sdk/types/src/types.input.ts

```diff
@@ -1201,7 +1201,7 @@ export interface AmbientLightSensor {
   /**
    * The ambient light in lux.
    */
-  ambientLight: number;
+  ambientLight?: number;
```

Comment:
@koush I noticed this type wasn't optional which seemed out of place, so I changed it.

Reply:
Yep you're correct